src/sif/golfcartcost.cc

```diff
@@ -0,0 +1,565 @@
+#include "sif/golfcartcost.h"
+#include "baldr/directededge.h"
+#include "baldr/graphconstants.h"
+#include "baldr/nodeinfo.h"
+#include "midgard/constants.h"
+#include "midgard/util.h"
+#include "proto_conversions.h"
+#include "sif/costconstants.h"
+#include "sif/osrm_car_duration.h"
+#include <cassert>
+
+// TODO: Inline tests like other profiles have
```

Comment:
This should have a question mark after it. I couldn't tell whether those inline tests were used and/or valuable, so I didn't write them when I first implemented the profile. Should I write some up before this gets merged (NOTE: there *are* Gurka tests)?

Reply:
the inline tests are the only way you can "unit test" the specific parsing functions because they are hidden in this implementation and not exposed in the headers. if you've tested all the stuff that this costing parses as costing options and feel that its covered in gurka there is no need to add them. if you look at other examples you'll see (iirc) that they check that the range objects that get setup are doing the right thing which is not too easy to test outside of this implementation file.

Reply:
Thanks, Kevin. I'll see about adding some quick tests. That said, I'm not too worried about it, and the Gurka tests actually do test the new parameter for max allowable road speed, and vehicle top speed isn't unique to this costing model.

Reply:
yeah tbh i wouldnt ask you to add the inline tests!